src/plugins/image/image.yml

```diff
@@ -49,3 +58,6 @@ orientationOptionHorizontal:
 smallImageSizeWarning:
   en: The selected image has a smaller width than the page content and might therefore be displayed smaller than expected.
   de: Das von Ihnen gewählte Bild hat eine geringere Breite als der Seiteninhalt und wird deshalb unter Umständen kleiner als erwartet angezeigt.
+smallClippingSizeWarning:
+  en: The selected clipping has a smaller width than the page content and might therefore be displayed smaller than expected.
+  de: Das von Ihnen gewählte Abschneidung hat eine geringere Breite als der Seiteninhalt und wird deshalb unter Umständen kleiner als erwartet angezeigt.
```

Comment:
Either "Der von Ihnen gewählte Zuschnitt hat ..." or "Der ausgewählte Bereich hat ..." (I'd like the second one better, also in English I'd say "The selected area has ...")

Reply:
Should we also change the above text in the same direction?
"Das von Ihnen gewählte Bild" -> "Das ausgewählte Bild"

Reply:
I like! (it sounds a little bit less accusing when we don't say 'your' or 'by you')

Reply:
Updated both cases!